Library/Formula/noti.rb

```diff
@@ -0,0 +1,56 @@
+require "language/go"
+
+class Noti < Formula
+  desc "Displays a notification after a terminal process finishes."
+  homepage "https://github.com/variadico/noti"
+  version "1.3.0"
+  head "https://github.com/variadico/noti.git"
+  if OS.mac?
+    url "https://github.com/variadico/noti/releases/download/v1.3.0/noti1.3.darwin-amd64.tar.gz"
```

Comment:
This needs to be built from source and we don't accept `OS.mac?` in Homebrew (Homebrew only supports OS X)